modules/nodes-pods-autoscaling-policies.adoc

```diff
@@ -0,0 +1,108 @@
+<2> Defines the scaling policy. 
+<3> The `type` parameter determines if the policy scales by a specific number of pods or a percentage of pods during each iteration. The default value is `pods`.
+<4> The `value` parameter determines the amount of scaling, either the number of pods or percentage of pods during each iteration. There is no default value for scaling down by number of pods.
+<5> The `periodSeconds` parameter determines the length of a scaling iteration. The default value is `15` seconds.
+<6> The default value for scaling down by percentage is 100%.
+<7>The `selectPolicy` parameter determines which policy to use first, if multiple policies are defined. Specify `Max` to use the policy that allows the highest amount of change, `Min` to use the policy that allows the lowest amount of change, or `Disabled` to prevent the HPA from scaling in that policy direction. The default value is `Max`.
+<8> The `stabilizationWindow` parameter determines the time period the HPA should look back at desired states. The default value is `0`.
+<9> This example creates a policy for scaling up.
+<10> The amount of scaling up by the number of pods. The default value for scaling up the number of pods is 4%.
+<11> The amount of scaling up by the percentage of pods. The default value for scaling up by percentage is 100%.
+
+The following example shows a scaling policy for scaling down:
+
+.Example policy for scaling down
+[source,yaml]
+----
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: hpa-resource-metrics-memory
+  namespace: default
+spec:
+...
+  minReplicas: 20
+...
+  behavior:
+    scaleDown:
+      stabilizationWindowSeconds: 300
+      policies:
+      - type: Pods
+        value: 4
+        periodSeconds: 30
+      - type: Percent
+        value: 10
+        periodSeconds: 60
+      selectPolicy: Max
+    scaleUp:
+      selectPolicy: Disabled
+----
+
+In this example, when the number of pods is greater than 40, the percent-based policy is used for scaling down, as that policy results in a larger change, as required by the `selectPolicy`. 
+
+Here, if there are 80 pod replicas, in the first iteration the HPA reduces the pods by 8, which is 10% of the 80 pods (based on the `type: Percent` and `value: 10` parameters), over one minute (`periodSeconds: 60`). For the next iteration, the number of pods is 72. The HPA calculates that 10% of the remaining pods is 7.2, which it rounds up to 8 and scales down 8 pods. On each subsequent iteration, the number of pods to be scaled is re-calculated based on the number of remaining pods. When the number of pods falls below 40, the pods-based policy is applied, because the pod-based number is greater than the percent-based number. The HPA reduces 4 pods at a time (`type: Pods` and `value: 4`), over 30 seconds (`periodSeconds: 30`), until there are 20 replicas remaining (`minReplicas`).
```

Comment:
```suggestion
If there are 80 pod replicas, the HPA reduces the pods by 8 in the first iteration, which is 10% of the 80 pods (based on the `type: Percent` and `value: 10` parameters), over one minute (`periodSeconds: 60`). For the next iteration, the number of pods is 72. The HPA calculates that 10% of the remaining pods is 7.2, which it rounds up to 8 and scales down 8 pods. On each subsequent iteration, the number of pods to be scaled is re-calculated based on the number of remaining pods. When the number of pods falls below 40, the pods-based policy is applied because the pod-based number is greater than the percent-based number. The HPA reduces 4 pods at a time (`type: Pods` and `value: 4`), over 30 seconds (`periodSeconds: 30`), until there are 20 replicas remaining (`minReplicas`).
```

Reply:
@codyhoag If I move _in the first iteration_, wouldn't the _which is 10%..._ grammatically apply to _iteration_, and not _8_?

Reply:
@mburke5678 Yeah, good call. The way you edited it in your latest commit sounds good to me :+1: